content/docs/concepts/security/index.md

```diff
@@ -461,10 +460,24 @@ peers:
   - mtls: {}
 {{< /text >}}
 
-Currently, the mutual TLS setting doesn't require any parameters. Hence,
-`-mtls: {}`, `- mtls:` or `- mtls: null` declarations are treated the same. In
-the future, the mutual TLS setting may carry arguments to provide different
-mutual TLS implementations.
+The mutual TLS setting has an optional `mode` parameter that defines the
+strictness of the peer transport authentication:
+
+- `mode: STRICT` enforces that all peers must use mutual TLS. This declaration
```

Comment:
Am I over-complicating things or duplicating information by putting this here? Should I just link further up to the section on strict vs permissive, link to the Policy reference docs or link to the mTLS migration guide?